scripts/ci/travis-test.sh

```diff
@@ -28,8 +28,9 @@ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make sanity-checks && xvfb-run --server-args=$XVFB_ARGS npm test'
+    -c 'make sanity-checks && xvfb-run --server-args=$XVFB_ARGS npm test && make installers-all'
 else
   make sanity-checks
   npm test
+  make installers-all
```

Comment:
I was thinking it probably ought to be in a separate `travis-build-installers.sh` script?

I just found it very odd that the TravisCI log said something like `scripts/ci/travis-test.sh exited with code 2`, and then scrolling up a bit further the error was actually coming from NPM.

\*shrug\* But if you think it's not worth it I guess the current approach works fine.